pnpm-lock.yaml

```diff
@@ -24,7 +24,7 @@ importers:
         version: 0.5.25
       '@brillout/test-types':
         specifier: ^0.1.10
-        version: 0.1.10(typescript@4.9.5)
+        version: 0.1.10(typescript@4.8.4)
```

Comment:
I don't understand why `cd docs && pnpm install @brillout/docpress@latest` changed the typescript version everywhere. I tried several times by doing `pnpm reset` just before but I couldn't prevent pnpm from touching these lines. Any ideas?

Reply:
I also noticed some behavior that seemed quite unexpected in that regard; I didn't dig into it yet.